style.css

```diff
@@ -125,4 +125,263 @@ body {
     align-items: center;
     list-style-type: none;
   }
+
+  .works {
+    display: grid;
+    grid-template-columns: repeat(1fr);
+    width: hug (375px);
+    height: hug (2, 788px);
+    top: 718px;
+    padding: 114px 24px 114px 24px;
+    gap: 88px;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border: solid #e5e5e5 1px;
+    background-color: #fff;
+    width: hug (327px);
+    height: hug (596px);
+    padding: 16px;
+    border-radius: 16px;
+    gap: 12px;
+  }
+
+  .cards-titles {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .subheading-lists {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 12px;
+    width: hug (236px);
+    height: hug (32px);
+  }
+
+  .canopy {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+    padding: 8px, 0, 8px, 0;
+  }
+
+  /* .point{
+    width: 8px;
+    height: 8px;
+    border-radius: 50%;
+    background: #C1C7D0;
+    color: #fff;
+
+  } */
```

Comment:
- [ ] Kindly remove commented code from your codebase for a more readable codebase.

Reply:
Okey
Thanks